src/model/NPMResolver.ts

```diff
@@ -323,4 +325,48 @@ export class NPMResolver {
     private getPackageLockPath(): string {
         return path.resolve(this.mainRepo, NPMResolver.PACKAGE_LOCK_JSON);
     }
+
+    private initRegistry(): void {
+        console.info("⟲ Initialising cplace jfrog registry for NPM");
+        const registry = '//cplace.jfrog.io/artifactory/api/npm/cplace-npm-local/';
+        const gradleProps: string = (execSync(path.join(this.mainRepo, 'gradlew properties')) || '').toString();
+
+        if (!gradleProps) {
```

Comment:
Do we want to stop the cplace-asc if anything fails here?

Reply:
NPM will fail later due to lack of creds. Unless the user has already logged in. Safe to continue.